src/patternfly/components/Button/button.scss

```diff
@@ -346,7 +362,8 @@
   transition-duration: var(--#{$button}--TransitionDuration);
   transition-property: var(--#{$button}--TransitionProperty);
 
-  &::after {
+  // Border pseudo-element
+  &::before {
```

Comment:
I'd be hesitant to make this change. Technically it isn't breaking, but given how common the button is and the number of overrides that likely exist, I could see this causing some styling issues. This may be the best way to make this change, so I'm not saying change it necessarily, but just want to mention that in case you can think of ways we could avoid swapping before/after styles. I can also noodle on it after reviewing this PR more thoroughly.